server/BREAKING.md

```diff
@@ -0,0 +1,31 @@
+
+## 0.1020 Breaking Changes
+
+- [`RestWrapper` is now an abstract class](#`restwrapper`-is-now-an-abstract-class)
+- [`Historian` no longer handles request headers](#`historian`-no-longer-handles-request-headers)
+
+### `RestWrapper` is now an abstract class
```

Comment:
You might want to explicitly call out that this is routerlicious (not historian)

Reply:
Should I add the package name (`@fluidframework/server-services-client`) next to the version number above? Both of these are routerlicious

Reply:
I think the confusion is the name "historian", cause its also the service name. So replacing with the package name sounds logical.